pennylane_lightning/src/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -66,6 +66,94 @@ class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
         return {parity_high, parity_middle, parity_low};
     }
 
+    static std::tuple<size_t, size_t, size_t, size_t>
+    revWireParity(size_t rev_wire0, size_t rev_wire1, size_t rev_wire2) {
+        using Util::fillLeadingOnes;
+        using Util::fillTrailingOnes;
+
+        size_t rev_wire_min = std::min(rev_wire0, rev_wire1);
+        size_t rev_wire_max = std::max(rev_wire0, rev_wire1);
+        size_t rev_wire_mid;
+
+        if (rev_wire2 < rev_wire_min) {
+            rev_wire_mid = rev_wire_min;
+            rev_wire_min = rev_wire2;
+        } else if (rev_wire2 > rev_wire_max) {
+            rev_wire_mid = rev_wire_max;
+            rev_wire_max = rev_wire2;
+        } else {
+            rev_wire_mid = rev_wire2;
+        }
+
+        size_t parity_low = fillTrailingOnes(rev_wire_min);
+        size_t parity_high = fillLeadingOnes(rev_wire_max + 1);
+        size_t parity_lmiddle =
+            fillLeadingOnes(rev_wire_min + 1) & fillTrailingOnes(rev_wire_mid);
+        size_t parity_hmiddle =
+            fillLeadingOnes(rev_wire_mid + 1) & fillTrailingOnes(rev_wire_max);
+
+        return {parity_high, parity_hmiddle, parity_lmiddle, parity_low};
+    }
+
+    static std::tuple<size_t, size_t, size_t, size_t, size_t>
+    revWireParity(size_t rev_wire0, size_t rev_wire1, size_t rev_wire2,
```

Comment:
Here too.

Reply:
Thanks @AmintorDusko ! Will do!

Reply:
Hi @multiphaseCFD, what you say makes sense, but the current docs say:
`/* Alias utility functions */`
I'm from the line of thought that believes that we must document as much as possible.
This will make future support and expansion more manageable and increase our chances of external contributions.